node/src/actors/chain_manager/mod.rs

```diff
@@ -334,9 +337,17 @@ impl ChainManager {
                     .consensus_constants
                     .mining_backup_factor;
                 let mut signatures_to_verify = vec![];
+                let mut vrf_input = self
```

Comment:
due to always that load the vrf_input, you change the checkpoint. Should be only saved the vrf_proof_hash in chain_info?

Reply:
I dont know I think it does not hurt to have it, specially since it is updated every time a block is consolidated. What do you guys think? In any case we would have to construct a beacon-like structure for the validations..

Reply:
Whatever is simpler to implement.

Reply:
Oks, simpler is to not change things xDD